node/actors/bft/src/replica/state_machine.rs

```diff
@@ -36,8 +41,11 @@ impl StateMachine {
     pub(crate) async fn start(
         ctx: &ctx::Ctx,
         config: Arc<Config>,
-        pipe: OutputSender,
-    ) -> ctx::Result<Self> {
+        outbound_pipe: OutputSender,
+    ) -> ctx::Result<(
+        Self,
+        sync::prunable_mpsc::Sender<Signed<ConsensusMsg>, ctx::Result<()>>,
```

Comment:
ditto: please document the result.

Reply:
Fixed, changes here: https://github.com/matter-labs/era-consensus/pull/56/commits/500b17f0ba75d3edebc77a04ae04d7de5a47fc5a